cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
   }
 
-  typeName(name) {
+  typeName() {
     this.nameField.type(name, { force: true });
   }
```

Comment:
The `typeName` method is missing the `name` parameter. You should add `name` as a parameter to this method to allow it to type the name into the name field.